cabby/geo/map_processing/map_structure.py

```diff
@@ -187,6 +196,18 @@ def write_map(self, dir_name: Text):
         else:
             map_logger.info("path {0} already exist.".format(path))
 
+        # Write edges.
+        path = self.get_valid_path(dir_name, '_edges', '.geojson')
+        if not os.path.exists(path):
+            # Drop columns with list type.
+            self.edges.drop(['highway', 'service', 'oneway', 'lanes', 'bridge',
+                             'access', 'est_width', 'name', 'maxspeed', 'ref'], 
+                             axis=1, inplace=True)
+            self.edges['osmid'] = self.edges['osmid'].apply(lambda x: str(x))
+            self.edges.to_file(path, driver='GeoJSON')
+        else:
+            map_logger.info("path {0} already exist.".format(path))
```

Comment:
path->Path, exist -> exists

Also, preferable to do map_logger.info(f'Path already exists: {path}')